deploy.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 #Suggest deploying to us-east-1 due to CE API, and SES
-export AWS_DEFAULT_REGION=us-east-1 
+export AWS_DEFAULT_REGION=us-east-1
+#Set to enable using a profile different than default ( aws configure --profile yourbillingprofile )
+export AWS_PROFILE=yourbillingprofile
```

Comment:
This line will stomp any preset AWS_PROFILE that's been set explicitly prior to invocation.